### ISSUE TITLE
.git folder is blocking other packages from being installed

### ISSUE THREAD

**Issue by ryanleecode** [kingjan1999/platform-folders#7]
After installing this package, other packages can't be installed because npm reports this package is a git repo.

```
npm ERR! git C:\Users\MyUser\Documents\MyProject\node_modules\platform-folders: Appears to be a git repo or submodule.
npm ERR! git     C:\Users\MyUser\Documents\MyProject\node_modules\platform-folders
npm ERR! git Refusing to remove it. Update manually,
npm ERR! git or move it out of the way first.
```

Removing the `.git` folder in `node_modules\platform-folders` seems to be a temporary fix that works.

**Comment by kingjan1999** [comment]
Thanks for the report! I've just published 0.2.7 which should exclude the folder. Please try and report back if that worked!

**Comment by ryanleecode** [comment]
Can you double check that the package was published successfully? Its still on 0.2.6 on the npm repository.

https://www.npmjs.com/package/platform-folders

**Comment by kingjan1999** [comment]
Oh, sorry, wrong registry. Should be published at npm now.
